src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.Win32Unix.cs

```diff
@@ -10,24 +10,24 @@ namespace System.IO.IsolatedStorage
 {
     internal static partial class Helper
     {
-        internal static string GetDataDirectory(IsolatedStorageScope scope, Environment.SpecialFolderOption option = Environment.SpecialFolderOption.None)
+        internal static string GetDataDirectory(IsolatedStorageScope scope)
         {
             // This is the relevant special folder for the given scope plus "IsolatedStorage".
             // It is meant to replicate the behavior of the VM ComIsolatedStorage::GetRootDir().
 
             // (note that Silverlight used "CoreIsolatedStorage" for a directory name and did not support machine scope)
 
             string dataDirectory = null;
-
+            var option = Environment.SpecialFolderOption.Create;
```

Comment:
Style nit, I think most people would just put the option directly in the call. But this is fine.

Reply:
And if it was going to be defined here, it should at least be a `const`.  But I suggest it be moved into the call sites.

Reply:
Thanks for the feedback.